frontend/common/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -11,41 +11,84 @@ export interface BreadcrumbsProps {
   links: BreadcrumbLink[];
 }
 
-const Breadcrumbs: React.FunctionComponent<BreadcrumbsProps> = ({ links }) => (
-  <div data-h2-display="base(flex)">
-    {links.map((link, index) => (
-      <Fragment key={link.title}>
-        {index > 0 && (
+const Breadcrumbs: React.FunctionComponent<BreadcrumbsProps> = ({ links }) => {
+  const MOBILE_WIDTH = 768;
+  const [isMobile, setIsMobile] = React.useState(false);
+
+  // choose the screen size
+  const handleResize = () => {
+    if (window.innerWidth < MOBILE_WIDTH) {
+      setIsMobile(true);
+    } else {
+      setIsMobile(false);
+    }
+  };
+  const previousStep = links[links.length - 2];
+
+  // create an event listener
+  React.useEffect(() => {
+    handleResize();
+
+    window.addEventListener("resize", handleResize);
+    return () => window.removeEventListener("resize", handleResize);
+  }, [isMobile]);
+
+  return (
+    <div data-h2-display="base(flex)">
+      {isMobile && previousStep ? (
+        <>
           <span
             data-h2-padding="base(0, x.25)"
             data-h2-display="base(flex)"
             data-h2-align-items="base(center)"
           >
-            <ChevronRightIcon style={{ width: "1.4rem" }} />
+            <ChevronLeftIcon style={{ width: "1.4rem" }} />
           </span>
-        )}
-        {link.href ? (
           <Link
             data-h2-display="base(flex)"
             data-h2-align-items="base(center)"
-            href={link.href}
-            key={link.title}
+            href={previousStep.href}
+            key={previousStep.title}
           >
-            {link.title}
+            {previousStep.icon || ""} {previousStep.title}
```

Comment:
We recently removed icons from breadcrumbs so i think we can just use `{link.title}` now

Reply:
Fixed by [Remove icon from breadcrumb link](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3678/commits/1fce5a392191f998122633106fb63454c14abde0)